tests/cupy_tests/test_cudnn.py

```diff
@@ -292,7 +292,7 @@ def setUp(self):
                 (self.groups > 1 and version < 7000)):
             self.err = ValueError
         elif deterministic and (
-                (self.dilate > 1 and (ndim != 2 or version < 7000)) or
+                (self.dilate > 1 and (ndim != 2 or version < 7300)) or
```

Comment:
Could you add some comments about what is used in each version of cuDNN, and why the test has conditional branches by the version of cuDNN?

Reply:
I fixed.

Reply:
It is a little difficult because it is determined by cuDNN inner specification.